tests/v2/validation/provisioning/ssh/ssh_test.go

```diff
@@ -0,0 +1,26 @@
+package ssh
+
+// create all tests here that use executable tests
+
+import (
+	"strconv"
+
+	"github.com/rancher/rancher/tests/framework/pkg/nodes"
+)
+
+// Test to check cpu usage
+
+func CheckCPU(node *nodes.Node) (string, error) {
+	command := "ps -C agent -o %cpu --no-header"
+	output, err := node.ExecuteCommand(command)
+	if err != nil {
+		return output, err
+	}
+
+	output_int, err := strconv.Atoi(output)
+	if output_int > 30 {
+		output = "ERROR: cluster agent cpu usage is too high. Current cpu usage is: " + output
```

Comment:
is this supposed to fail the test? from what I see, this would still pass, though the language around it does not suggest so.